File: investment_tracker_be/investment_tracker_be/Models/TrackerLookup.cs
using System;
using System.Collections.Generic;

namespace investment_tracker_be.Models
{
    public partial class TrackerLookup
    {
        public TrackerLookup()
        {
            InvestmentFundLogCurrencies = new HashSet<InvestmentFundLog>();
            InvestmentFundLogStatuses = new HashSet<InvestmentFundLog>();
        }

        public int LookupId { get; set; }
        public string? LookupType { get; set; }
        public string? LookupDescription { get; set; }
        public string? HiddenValue { get; set; }
        public string? VisibleValue { get; set; }
        public bool IsActive { get; set; }
        public DateTime? Createddate { get; set; }

        public virtual ICollection<InvestmentFundLog> InvestmentFundLogCurrencies { get; set; }
        public virtual ICollection<InvestmentFundLog> InvestmentFundLogStatuses { get; set; }
    }
}

File: investment_tracker_fe/src/app/shared/models/trackerLookup.ts
import { InvestmentFundLog } from "./investmentFundLog";

export interface trackerLookup {
    lookupId: number;
    lookupType: string | null;
    lookupDescription: string | null;
    hiddenValue: string | null;
    visibleValue: string | null;
    isActive: boolean;
    createddate: string | null;
    investmentFundLogCurrencies: InvestmentFundLog[];
    investmentFundLogStatuses: InvestmentFundLog[];
}